### ISSUE TITLE
gather window?

### ISSUE THREAD

**Issue by theHankTaylor** [nicolewhite/same-domain-tabs#2]
idk if this is possible but if there was a button to consolidate all chrome tabs into one window, that'd be amazing.

**Comment by nicolewhite** [comment]
yeah this should be possible according to the chrome.tabs.move docs

https://developer.chrome.com/extensions/tabs#method-move

i'm thinking i'll add versions of both close and gather that work across all windows as opposed to only the active one
